### ISSUE TITLE
Confusion in Snake/Camel Case Plugin Argument

### ISSUE THREAD

**Issue by zusamann** [graphql-python/graphene#244]
Graphene's Default Behavior is something like this:

Query:
```
{
    jobs(id: "123918239", jobId: 2) {
        ...
        ...
    }
}
```
```
for i in args:
    print i

>> id
>> job_id
```

With the plugin argument of:

```
schema = graphene.Schema(
    query=QueryType,
    auto_camelcase=False
)
```
```
for i in args:
    print i

>> id
>> jobId
```

Now isn't the plugin argument naming erroneous?
Printing `snake_case` by default is fine. But not printing `snake_case` when `auto_camelcase=False` is confusing. It should be `auto_snakecase=False` that stops the automatic `snake_casing`.

**Comment by syrusakbary** [comment]
This will be fixed in `1.0.dev` once we support disabling `auto_snakecase` (soon).

**Comment by syrusakbary** [comment]
This is now fixed in `master` with the commit: https://github.com/graphql-python/graphene/commit/9f30aa2d45761aa29b03585adde8ff556f3a3ba3

(You can update graphene with `pip install graphene>=1.0.dev`).
